emcc.py

```diff
@@ -84,7 +84,7 @@
     '--bind', '--closure', '--cpuprofiler', '--embed-file',
     '--emit-symbol-map', '--emrun', '--exclude-file', '--extern-post-js',
     '--extern-pre-js', '--ignore-dynamic-linking', '--js-library',
-    '--js-transform', '--memory-init-file', '--oformat', '--output_eol',
+    '--js-transform', '--memory-init-file', '--oformat', '--output_eol', '--output-eol',
```

Comment:
Why support both `_` and `-`?

Should this go in the changelog?

Reply:
I'm renaming the option to be consistent with all our other ones and keeping the legacy name in case anyone is using it

Reply:
Sounds good. Please note in the changelog.